src/EditorFeatures/CSharpTest/GenerateFromMembers/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersTests.cs

```diff
@@ -642,5 +642,46 @@ public C(int i, int hello)
 }"
             );
         }
+
+        [WorkItem(33602, "https://github.com/dotnet/roslyn/issues/33602")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddConstructorParametersFromMembers)]
+        public async Task TestConstructorWithNoParameters()
```

Comment:
Can we also have a test with only implicit default ctor?

Reply:
Yes, we have this test:
```csharp
class C
{
    [|int i;|]
    int Hello { get; set; }
}
`